apps/scratch/src/pages/tests/Edit.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import { readFileSync } from "fs";
+import { resolve } from "path";
+import { test, expect } from "playwright-test-coverage";
+
+// eslint-disable-next-line no-undef
+const testTask = readFileSync(resolve(__dirname, "test-task.zip"));
+
+declare global {
+  interface Window {
+    postedMessages: { message: unknown; options: unknown }[];
+  }
+}
```

Comment:
The intention is clear from the tests, but this could benefit from a little documentation, since it's central to how the tests work.